source/boards/work-with-cards.rst

```diff
@@ -18,7 +18,101 @@ Work with Cards
+
+Share card previews
+-------------------
+
+When you share a link to a card within Mattermost, the card details are automatically displayed in a preview. This preview highlights what the card is about at a glance without having to navigate to it.
+
+Customize card properties
+-------------------------
+
+Cards can contain different data fields depending on the purpose of the board. For example, in a **Roadmap** board, cards include a **Type** field where you can add categories such as **Bug**, **Triage**, etc. In a **Task** board, cards include **Status** and **Priority** fields instead.
+
+Add and manage properties
+-------------------------
+
+To create a new a new property field open a card and select **Add a property**. Then select the type of property from the drop-down menu. The property type specifies the type of data you plan to capture within that field. When you create new card properties, they are added to all new and all existing cards on the current board.
+
+Properties are automatically added to the board filter list at the top of the page, so ensure you customize all property names to make it easy to filter your board by specific properties later. To rename a property, open a card and select the property name to open an editable field. Your changes are saved immediately, and applied across all cards on the current board.
+
+To delete properties you no longer need, select the property, then choose **Delete**. You'll be asked to confirm that you want to remove that property from every card on the current board.
+
+Once you have card properties defined, you have full control over which properties are visible on the board. Select **Properties** at the top of the board, then enable all properties you want to see at a glance, and hide all properties you don't want to see.
+
+Work with property types
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+* **Checkboxes** are useful for agenda items that are regularly revisited in weekly sprint or grooming meetings.
+* **Multi-select** allows you to create badges to indicate things like status.
+* **Numbers** are useful to capture metrics such as task sizing or effort estimates.
+* **Person** provides a quick way to capture user assignments.
+* **URL** can be used to provide a link to a pull request or relevant website.
+* **Created time/Created by/Last updated time/Last updated by** are good ways order tasks in order of recency.
+
+Calculations
+------------
+
+When you view a board in table view, you can use calculations to answer basic metric questions without needing to create complex reports. Hover over the bottom of a column to display the **Calculate** feature, then select the arrow to open the menu options.
```

Comment:
> When you view a board in table view, you can use calculations to answer basic metric questions

This sentence hints that the `Calculations` feature is for table view only, but it's also for Kanban view/Board view as we've already mentioned in the docs below.